output/rhv-upload-plugin.py

```diff
@@ -25,7 +25,7 @@
 
 from contextlib import contextmanager
 from http.client import HTTPSConnection, HTTPConnection
-from urllib.parse import urlparse
+from urllib.parse import urlparse, urlunparse
```

Comment:
We don't need urlunparse() in this module, please revert this change.

This module uses ovirt-imageio url which cannot have a username in the netloc.